src/ui/Controls/VideoPlayerContainer.cs

```diff
@@ -399,6 +400,30 @@ public void SetSubtitleText(string text, Paragraph p, Subtitle subtitle)
             }
         }
 
+        private SsaStyle _mpvStyle;
```

Comment:
It would be a little better if it was an "_mpvStyleHeader", so generating the header will happen less often.